third_party/terraform/utils/iam_kms_key_ring.go.erb

```diff
@@ -48,7 +49,11 @@ func KeyRingIdParseFunc(d *schema.ResourceData, config *Config) error {
 }
 
 func (u *KmsKeyRingIamUpdater) GetResourceIamPolicy() (*cloudresourcemanager.Policy, error) {
+<% if version == 'ga' -%>
 	p, err := u.Config.clientKms.Projects.Locations.KeyRings.GetIamPolicy(u.resourceId).Do()
+<% else -%>
+	p, err := u.Config.clientKms.Projects.Locations.KeyRings.GetIamPolicy(u.resourceId).OptionsRequestedPolicyVersion(iamPolicyVersion).Do()
```

Comment:
Do we not need a similar change to the `SetIamPolicy` calls? Maybe not because the policy would have the policy version attached by that point?

Reply:
It's already there in the resource_iam_*.go files